functions/source/update_custommanaged_cmk_policy.py

```diff
@@ -0,0 +1,167 @@
+import json
+import logging
+import threading
+import boto3
+import cfnresponse
+
+# boto client
+
+client = boto3.client('kms')
+
+databricks_s3bucketonly_statement1 = {
+  "Sid": "Allow Databricks to use KMS key for DBFS",
+  "Effect": "Allow",
+  "Principal":{
+    "AWS":"arn:aws:iam::414351767826:root"
+  },
+  "Action": [
+    "kms:Encrypt",
+    "kms:Decrypt",
+    "kms:ReEncrypt*",
+    "kms:GenerateDataKey*",
+    "kms:DescribeKey"
+  ],
+  "Resource": "*"
```

Comment:
Can we restrict to specific resources?

Reply:
These actions are per our documentation [https://docs.databricks.com/security/keys/customer-managed-keys-storage-aws.html#step-1-create-or-select-a-key]. Once the policy is built, the customer can edit the policy to restrict the resources.